src/components/EstimateComboBox.tsx

```diff
@@ -37,10 +37,15 @@ interface EstimateComboBoxProps {
     onChange?: (estimate?: { date: string; q: string; y: string }) => void;
 }
 
-const StyledInput = styled(Input)`
-    min-width: 100px;
+const StyledInput = styled(Input)<{ width?: number }>`
```

Comment:
Input has prop `brick`, u can use it like `<Input brick="right" ... />`. Button has same prop ;)